### ISSUE TITLE
[CONTACT] help with user account

### ISSUE THREAD

**Issue by ccicero** [ArctosDB/arctos#5235]
@dustymc I'm trying to resurrect/create a permission-level account for Elizabeth J. Beckman (aka Libby Beckman) and am running into issues.

-- Libby has two accounts currently, under usernames 'libby' and 'lbeckman'
-- Agent table has the login 'libby'
-- Both accounts were locked and I tried unlocking them with password reset
-- Libby got the passwords by email but when trying to login with those Arctos-issued passwords got an 'authentication failed' error
-- Account under 'libby' says it's active and open but she can't login and needs a password change but that's not working
-- Then we tried creating a new account from scratch under 'ejbeckman' but can't add that login name to her agent record because she already has a login name type, and we can't delete/edit the existing login name type.

Ideally her active account would be under 'lbeckman' so how do we update the agent record with that new login and reset her password so that she can authenticate?

Help!?!

**Comment by dustymc** [comment]
https://arctos.database.museum/ChangePassword.cfm?action=lostPass

enter the credentials from https://arctos.database.museum/AdminUsers.cfm?action=edit&username=lbeckman, wait for the email

**Comment by ccicero** [comment]
@dustymc Tried that but here's the problem:

lbeckman username says 'Agent not found'
In the agent record, the login name = 'libby'
I can't add 'lbeckman' as another login name ---> get an error that the username already exists

![image](https://user-images.githubusercontent.com/967275/199580761-219589b2-f2e0-4086-bfcf-d2ddc26b3f4b.png)

So not sure where to go from here. It might be easier to do this by screen sharing through zoom.

**Comment by dustymc** [comment]
Oh derp sorry I skimmed over the permission-level account - I think you mean Operator.

Agents must be unique, and each can have one Operator account. The ONLY thing that can work is to resurrect libby.

That all looks happy, but there were a bunch of password reset requests and something probably got mixed up. If you have the emails, use the one where the password starts with 'K' (its most recent). If you don't let me know and I can forward, or Michelle has signed up to get the logs and will be ccd on that.

**Comment by ccicero** [comment]
@dustymc We tried different passwords sent from Arctos and got the 'Authentication failed' error but I'll check with Libby and we can try again with the one that starts with 'K'

Will let you know if we get in (or not).

Thanks.

**Comment by dustymc** [comment]
>'Authentication failed'

AHA!

https://handbook.arctosdb.org/documentation/users.html

The COLDFUSION_USER role must be provided for the user to log in. 

Grant some roles, then try logging in

**Comment by ccicero** [comment]
now I can't access Arctos at all - did it just go down?

**Comment by ccicero** [comment]
503 Service Not Available

**Comment by Jegelewicz** [comment]
It's back now.

**Comment by ccicero** [comment]
it's back!

**Comment by dustymc** [comment]
happy?
